lineapy/db/db.py

```diff
@@ -85,6 +85,20 @@ def __init__(self, url: str):
         self.session.configure(bind=self.engine)
         Base.metadata.create_all(self.engine)
 
+        # establish alembic db version for migration
+        # https://alembic.sqlalchemy.org/en/latest/cookbook.html#building-an-up-to-date-database-from-scratch
+        from alembic import command
+        from alembic.config import Config
+
+        lp_install_dir = Path(__file__).resolve().parent.parent
+
+        alembic_cfg = Config(lp_install_dir / "alembic.ini")
+        alembic_cfg.set_main_option(
+            "script_location", (lp_install_dir / "_alembic").as_posix()
+        )
+        alembic_cfg.set_main_option("sqlalchemy.url", self.url)
```

Comment:
this bit is almost exactly the same as the one that is called in cli/cli.py . suggest that this be pulled out into a function that takes in arg for init vs update (which will decide whether `stamp` is called or `upgrade` is)